### PR TITLE
Part: Filter selection GUI texts to sentence case

### PR DESCRIPTION
**Pull request by Roy-043**

### CODE REVIEW

File: src/Mod/Part/Gui/CommandFilter.cpp [hunk 1]
```diff
@@ -170,10 +170,10 @@ PartCmdVertexSelection::PartCmdVertexSelection()
   : Command("Part_VertexSelection")
 {
     sGroup        = "Standard-View";
-    sMenuText     = QT_TR_NOOP("Vertex Selection");
-    sToolTipText  = QT_TR_NOOP("Select a Vertex/Vertices");
+    sMenuText     = QT_TR_NOOP("Vertex selection");
+    sToolTipText  = QT_TR_NOOP("Select a vertex/vertices");
```
Comment (wwmayer) [inline src/Mod/Part/Gui/CommandFilter.cpp:174]: Should we drop the singular form? I think if it says "Select vertices" then it should be clear that one can also select a single vertex only.

**Comment by Roy-043** [comment]
Maybe go for a longer sentence:
"Only allow the selection of vertices"
(and change the other tooltips accordingly)

**Comment by wwmayer** [comment]
Would be good for me too

**Review by chennes — APPROVED** [review]
These changes look good to me.

**Comment by wwmayer** [comment]
For me too.

**Pull request merged**